src/gravel/controllers/nodes/mgr.py

```diff
@@ -139,7 +139,7 @@ def __init__(self):
         self._node_init()
         assert self._state
 
-        logger.debug(f"=> mgr -- init > {self._state}")
+        logger.debug(f"init > {self._state}")
```

Comment:
when logging data, it would be great to reader see what is it, in this case some info that this is a state be enough I guess.

Reply:
Believe me, when it shows in the log you know it's state ;)

Reply:
I am fine with that.

Reply:
It is non-blocking comment, feel free to merge when ready.